File: JAVA Projects/linked_list/Single_ll.java
package linked_list;

public class Single_ll {
    public static class Node{
        private int value;
        void SetValue(int value){
            this.value = value;
        }
        public Node next;
    }

    public static void main(String[] args) {
        Node head= new Node();
        head.SetValue(1);
        head.next = new Node();
        head.next.SetValue(2);
        head.next.next=new Node();
        head.next.next.SetValue(3);
    }
}

File: C++ Projects/DSA/LinkedList/SingleLL.cpp
#include <iostream>
using namespace std;

class Node {
    private: int value;
    public: Node(int val){
        value=val;
    }
    public: Node* next;
};

int main(){
    Node head = Node(1);
    head.next=new Node(2);
    head.next->next = new Node(3);
    head.next->next->next = new Node(4); 
}

